department-of-veterans-affairs/abd-vro
# MCP-1897 Save evidence summary document to db

## What was the problem?
We need to add a record for evidence summary documents in the db after the generate PDF  service.

Associated tickets or Slack threads:

- [MCP-1897](https://amida.atlassian.net/browse/MCP-1897)

## How does this fix it?[^1]


## How to test this PR
- Run full health assessment endpoint
- generate a pdf
- check database for evidence summary document


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
